src/app/pages/ParaTimesPage/TransactionRecipient/__tests__/index.test.tsx

```diff
@@ -166,33 +166,17 @@ describe('<TransactionRecipient />', () => {
     expect(navigateToAmount).toHaveBeenCalled()
   })
 
-  it('should navigate back to deposit', async () => {
-    const navigateToDeposit = jest.fn()
-    jest.mocked(useParaTimesNavigation).mockReturnValue({
-      ...mockUseParaTimesNavigationResult,
-      navigateToDeposit,
-    })
-    render(<TransactionRecipient />)
-
-    await userEvent.click(screen.getByRole('button', { name: 'Previous' }))
-
-    expect(navigateToDeposit).toHaveBeenCalled()
-  })
-
-  it('should navigate back to withdraw', async () => {
-    const navigateToWithdraw = jest.fn()
+  it('should cancel transfer', async () => {
+    const clearTransactionForm = jest.fn()
     jest.mocked(useParaTimes).mockReturnValue({
       ...mockUseParaTimesResult,
+      clearTransactionForm,
       isDepositing: false,
     } as ParaTimesHook)
-    jest.mocked(useParaTimesNavigation).mockReturnValue({
-      ...mockUseParaTimesNavigationResult,
-      navigateToWithdraw,
-    })
     render(<TransactionRecipient />)
 
-    await userEvent.click(screen.getByRole('button', { name: 'Previous' }))
+    await userEvent.click(screen.getByRole('button', { name: 'Cancel transfer' }))
 
-    expect(navigateToWithdraw).toHaveBeenCalled()
+    expect(clearTransactionForm).toHaveBeenCalled()
```

Comment:
could we instead have a test more like

> sample: fill in Cipher recipient go back to the first step, select emerald, click next and we have Oasis addr in recipient input where we need eth addr.

Reply:
That belongs to e2e test more. I can add one and comment it out temporary. Which one is now preferable in Wallet: cypress or playwright  ?

Reply:
playwright